.travis.yml

```diff
@@ -10,20 +10,23 @@ addons:
     packages:
       - libpulse-dev
       - pulseaudio
+      - pulseaudio-utils
       - dbus-x11
-      - gstreamer1.0-plugins-base
+services:
+  - pulseaudio
 
 before_install:
-  - curl -L https://github.com/ferjm/gstreamer-1.14.1-ubuntu-trusty/raw/master/gstreamer.tar.gz | tar xz
-  - sed -i "s;prefix=/root/gstreamer;prefix=$PWD/gstreamer;g" $PWD/gstreamer/lib/x86_64-linux-gnu/pkgconfig/*.pc
-  - export PKG_CONFIG_PATH=$PWD/gstreamer/lib/x86_64-linux-gnu/pkgconfig
-  - export GST_PLUGIN_SYSTEM_PATH=$GST_PLUGIN_SYSTEM_PATH:/usr/lib/x86_64-linux-gnu/gstreamer-1.0:$PWD/gstreamer/lib/x86_64-linux-gnu/gstreamer-1.0
-  - export GST_PLUGIN_SCANNER=$PWD/gstreamer/libexec/gstreamer-1.0/gst-plugin-scanner
-  - export PATH=$PATH:$PWD/gstreamer/bin
-  - export LD_LIBRARY_PATH=$PWD/gstreamer/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH
+  - curl -L https://github.com/ferjm/gstreamer-1.14.1-ubuntu-trusty/raw/master/gstreamer_.tar.gz | tar xz
```

Comment:
You probably want to document in there how you built it so that other people can reproduce it later :)